### DISCUSSION TITLE
使用期间微信提示系统环境异常

### DISCUSSION THREAD

**Discussion by fanbili** [huiyadanli/RevokeMsgPatcher#681] [General]
RT，微信版本3.9.9.27，试用期间突然弹出对话框提示什么系统环境异常，当时忘记截图了，紧接着重新登陆后微信团队发来消息如图，
![微信截图_20240124105814](https://github.com/huiyadanli/RevokeMsgPatcher/assets/17902403/55a06aa2-72eb-4559-b1f7-f22405b1c4fc)

**Comment by huiyadanli** [comment]
补丁一般不会引发这类问题，请检查你自身系统环境。

参考 #666
